guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
 #### Resolved Issues
 
+-  <!--MAGECLOUD-2736-->Resolved an issue preventing a successful installation of Magento when specifying a custom database configuration.
+
+-  <!--MAGECLOUD-2740-->Fixed an issue that neglected to include each website locale in the `systems` section of the `config.local.php` file after running the `config:dump` command.
```

Comment:
Consider: Fixed an issue with the `config:dump` CLI command to ensure that it writes website locale data to the `systems` section of the `config.local.php` file.